src/main/java/com/cag/cagbackendapi/services/user/impl/UserService.java

```diff
@@ -36,4 +40,27 @@ public UserResponseDto registerUser(RegisterUserRequestDto registerUserRequestDt
 
         return userDao.saveUser(registerUserRequestDto);
     }
+
+    public UserResponseDto getByUserId(String userId) {
+
+        if(userId == null || userId == "") {
+            throw new BadRequestException("Invalid userId", null);
```

Comment:
You can put the "Invalid User Id." message into the constants file in the DetailedErrorMessages object and reference it here as:DetailedErrorMessages.INVALID_USER_ID